src/components/NavBar.js

```diff
@@ -0,0 +1,15 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+
+const Navigation = () => (
+  <nav>
+    <h1>BookStore</h1>
+    <ul>
+      <li><Link to="/">BOOKS</Link></li>
+      <li><Link to="/category">CATEGORY</Link></li>
+    </ul>
+    <p>profile icon</p>
+  </nav>
```

Comment:
- [ ] Hello you have no styling on this project you submitted kindly read the project requirement before you start the project so as to guide you on what to do, based on the project description You will find all the details of the Bookstore design in [Zeplin](https://zeplin.io/). [Log in to Zeplin](https://app.zeplin.io/login) (hint: you will find the credentials in the [sneak peek](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md) of the project) and open the project Bookstore CMS.
- [ ] Kindly follow the instructions on the project requirement to complete this milestone. according to project requirements, Your final design should match Zeplin's design (this includes all elements + the same styling)
You may use the CSS rules detailed in Zeplin to style the website.

Your design should look like this based on the requirement 

![bookstore](https://github.com/obibaadoma/book-stores/assets/92458236/62c1701c-94b2-4ad2-b791-d2e999c9877a)

- [ ] kindly add the style to your website and resubmit for a new review to make your website look professional user friendly

Reply:
notead have included ziplin